management_service/package-lock.json

```diff
@@ -22,6 +22,7 @@
         "nodemon": "^3.1.9",
         "prettier": "^3.5.2",
         "prettier-eslint": "^16.3.0",
+        "tsc-alias": "^1.8.11",
```

Comment:
this should be removed.

Reply:
Might be best to just revert this file completely since we didn't actually want to change anything here. Its not harmful though.